src/frontend/components/UI/SearchBar/index.css

```diff
@@ -25,6 +25,7 @@
   padding: var(--space-xs) var(--space-md);
   text-align: left;
   overflow-x: hidden;
+  z-index: 1;
```

Comment:
this z-index is not really needed, it's enough with the z-index: 10 in the Header css

Reply:
Nope this is needed as well, otherwise the filters go on top of the search suggestions